docs/intro/tutorial01.txt

```diff
@@ -287,12 +287,12 @@ an :func:`~django.conf.urls.include` in the ``urlpatterns`` list, so you have:
 .. snippet::
     :filename: mysite/urls.py
 
-    from django.conf.urls import include, url
+    from django.urls import include, path
```

Comment:
Currently `include` is in `django.conf.urls` still. Going to fix that in the implementation PR.